pkgs/applications/misc/alvr/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib,
```

Comment:
This package currently can't be built, because it's not added to `pkgs/top-level/all-packages.nix`. Alternatively, you could consider putting it in `pkgs/by-name`

Reply:
Oh whoops